### PR TITLE
Add LST Backward Average script

### PR DESCRIPTION
**Pull request by AmberMulder**
Add documentation and evalscripts on computation of Land Surface Temperature Backward Average.

### CODE REVIEW

**Comment by AmberMulder** [comment]
@jonasViehweger Can you please review?

File: planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/raw.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+//VERSION=3
+
+// LST has two observations per day: 1h30 and 13h30 solar local time
+
+const date = "2022-12-31"; // The date for which the backward average is calculated
```
Comment (jonasViehweger) [inline planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/raw.js:5]: Compared to the same script for SWC, why is this needed?

File: planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/script.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+//VERSION=3
+
+// Set defaultVis to false to scale and set color_min and color_max values.
+// LST has two observations per day: 1h30 and 13h30 solar local time
+
+const date = "2022-12-31"; // The date for which the backward average is calculated
+const nDays = 20; // The number of days to load data for
+const scaleFactor = 100; // The scale factor for the SWC values
+const defaultVis = true; // If true, the default visualization will be applied. If false, the color_min and color_max values will be used.
```
Comment (jonasViehweger) [inline planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/script.js:9]: Here it's the same as for SWC, we can now set the min/max directly in the ColorRamp, so no need to do it ourselves.

```suggestion
```

File: planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/script.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  for (let i = 0; i < numIntervals; i++) {
+    cmap[i][0] = min + intervalLength * i;
+  }
+}
+
+// Update the min max of color bar if defaultVis set to false
+if (!defaultVis) updateCMap(color_min, color_max);
+
+// Initialize the ColorRamp
+const visualizer = new ColorRampVisualizer(cmap);
```
Comment (jonasViehweger) [inline planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/script.js:94]:
```suggestion
// Initialize the ColorRamp
const visualizer = new ColorRampVisualizer(cmap, color_min, color_max);
```

**Review by jonasViehweger — COMMENTED** [review]
One question about the `calculationDate`, mainly why this is done in the script and not through setting the process API request to a different end date.

Other than that looks good!

File: planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/raw.js (hunk 1 above)
Comment (AmberMulder) [reply planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/raw.js:5]: For LST we have both daytime and nighttime measurements in the same collection and therefore need to use `mosaicking : "TILE"` rather than `ORBIT` to be able to select the data of interest. ATM it is [not possible](https://docs.sentinel-hub.com/api/latest/evalscript/v3/#collections) to extract the values for  `timeRange.from` and `timeRange.to` (defined in the body of the request) in a V3 EvalScript when using mosaicking `TILE`. This is something the SH team is now aware of, but for now, I needed to find an alternative way to fix the calculation date.

File: planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/script.js (hunk 2 above)
Comment (AmberMulder) [reply planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/script.js]: Initially I did not add this to be able to have a `defaultVis` just like the other LST custom script examples. But I agree it would be better to immediately include the update in the `ColorRampVisualizer` initialization. Therefore, I have updated it.

File: planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/script.js (hunk 3 above)
Comment (AmberMulder) [reply planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/script.js]: See comment above. Updated.

**Comment by AmberMulder** [comment]
@jonasViehweger Thanks for the review, I have added the changes!

File: planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/raw.js (hunk 1 above)
Comment (jonasViehweger) [reply planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/raw.js:5]: Ah, makes sense!
